pkg/webreg/webreg.go

```diff
@@ -729,6 +729,78 @@ to configure that test to run on a schedule, instead of as a pre-submit:
+    <td>Images composing the <code>release:name</code> release payload, present when <code>&lt;name&gt;<code> is configured in <code>releases<code>.</td>
+  </tr>
+  <tr>
+    <td style="white-space: nowrap"><code>stable<code></td>
+    <td>Same as above, but for the <code>release:latest</code> release payload. Appropriate tags are overridden using the container images built during the test.</td>
+  </tr>
+</table>
+
+<h4 id="config-references"><a href="#config-references">Referring to Images in <code>ci-operator</code> Configuration</a></h4>
+<p>
+Inside of any <code>ci-operator</code> configuration file all images must be 
+referenced as an <code>ImageStreamTag</code> (<code>stream:tag</code>), but
+may be referenced simply with the tag name. When an image is referenced with
+a tag name, the tag will be resolved on the <code>pipeline</code> <code>ImageStream</code>,
+if possible, falling back to the <code>stable</code> <code>ImageStream</code>
+if not. For example, an image referenced as <code>installer</code> will use
+<code>pipeline:installer</code> if that tag is present, falling back to
+<code>stable:installer</code> if not. The following configuration fields
+use this defaulting mechanism:
+</p>
+
+<ul>
+  <li><code>images[*].from</code>: configuring the base <code>FROM</code> which an image builds</li>
+  <li><code>promotion.additional_images</code>: configuring which images are published</li>
+  <li><code>promotion.excluded_images</code>: configuring which images are not published</li>
+  <li><code>tests[*].container.from</code>: configuring the container image in which a single-stage test runs</li>
+  <li><code>tests[*].steps.{pre,test,post}[*].from</code>: configuring the container image which some part of a multi-stage test runs</li>
+</ul>
+
+<h4 id="literal-references"><a href="#literal-references">Referring to Images in Tests</a></h4>
+<p>
+In some cases, it is necessary for a test command to refer to an image that
+was built during the test workflow. Test workloads can declare that they require
+fully resolved pull specification as a digest for any image from the <code>pipeline</code>,
+<code>stable-&lt;name&gt;</code> or <code>release</code> <code>ImageStreams</code>. 
+Tests may opt into having these environment variables present by declaring
+<code>dependencies</code> in the <code>ci-operator</code> configuration for the
+test. For instance, the example container test will be able to access the following
+environment variables:
+</p>
+
+<ul>
+  <li><code>${MACHINE_CONFIG_OPERATOR}</code>: exposing an the pull specification of the <code>stable:machine-config-operator</code> <code>ImageStreamTag</code></li>
+  <li><code>${BINARIES}</code>: exposing an the pull specification of the <code>pipeline:test-bin</code> <code>ImageStreamTag</code></li>
+  <li><code>${LATEST_RELEASE}</code>: exposing an the pull specification of the <code>release:latest</code> release payload <code>ImageStreamTag</code></li>
```

Comment:
IMHO `s#<code>release:latest</code> release payload#<code>release:latest</code> payload#` so it reads less stuttery

Reply:
There's also an extra "an" in each item.